DEP Tasks/Trainers Task/JSCore/YoutubeDummy/index.js

```diff
@@ -0,0 +1,81 @@
+//AIzaSyAkNkBT_H-jfF3J8jIB083yonYXrQKMUBo
+
+var searchBox = document.getElementById('searchInput');
```

Comment:
Use `const` instead of `var`. Same for all variables in this files

Reply:
replaced var to const or let as per as required